### ISSUE TITLE
Is it possible to render a SliverList inside a SliverList?

### ISSUE THREAD

**Issue by kevin4dhd** [Kavantix/sliver_tools#42]
Hello, I'm a bit new to this topic of silvers, I'm looking for the following:
I have a CustomScrollView, where its children are a SliverAppBar and a SliverList, I want another SliverList to exist inside the SliverList.
I have tried to do it with this library and without it and neither of the two ways works for me.

![image](https://user-images.githubusercontent.com/22886367/155435234-d87daffe-49f6-4d5e-a2d6-3aff132f1be4.png)

In summary I need to create several big lists within another list, I don't want to use shrinkwrap because my application becomes really slow.

**Comment by Kavantix** [comment]
Nesting sliver lists is not possible at the moment.
A completely custom sliver would need to be built to support something like this.

For reference, what are the sizes of your outer and inner lists?

**Comment by kevin4dhd** [comment]
The size of the outer list can be from 1 to 900, the inner list can be from 1 to 300

**Comment by Kavantix** [comment]
So the list is possibly 270000 items long?
Is the idea that the user scrolls to all of this or do they need to be scrollable on their own?
